tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_LocalReturn.js

```diff
@@ -11,7 +11,7 @@ function /*[#|*/f/*|]*/() {
 async function f() {
     let x = fetch("https://typescriptlang.org").then(res => console.log(res));
     try {
-        return x;
+        return await x;
```

Comment:
Not if you're in a `try/catch`. If you `return` the promise and it rejects, the `catch` will never be triggered because you've already exited the function.